pykafka/balancedconsumer.py

```diff
@@ -261,7 +271,7 @@ def _setup_zookeeper(self, zookeeper_connect, timeout):
         self._zookeeper = KazooClient(zookeeper_connect, timeout=timeout / 1000)
         self._zookeeper.start()
 
-    def _setup_internal_consumer(self, start=True):
+    def _setup_internal_consumer(self, partitions=[], start=True):
```

Comment:
We shouldn't use mutable kwargs - I think `partitions=None` and some logic in the function to set it to `[]` in that case should do the trick. I can add that.